pages/pools/index.tsx

```diff
@@ -48,8 +49,18 @@ const Pools: FC = () => {
 		Number(router.query.page ?? DEFAULT_PAGE_INDEX)
 	);
 
-	const poolsQuery = useGetPoolsQuery({ networkId: network.id });
+	const poolsQuery = useGetPoolsQuery();
+	const poolsQueryWithNetwork = useMemo(() => {
+		return poolsQuery
+			.filter((q) => !!q.data)
+			.reduce((prev, current) => {
```

Comment:
I'm confused. What's the goal of `reduce` here?

Reply:
it is to add the network name to each row. It is ok

Reply:
that's correct. It's to add the network name to each poolQuery result object